.github/workflows/rust.yml

```diff
@@ -52,8 +52,5 @@ jobs:
       - name: Enforce formatting
         run: cargo fmt -- --check
 
-      # - name: Lint (clippy)
-      #   uses: actions-rs/cargo@v1
-      #   with:
-      #     command: clippy
-      #     args: --all-features -- -D warnings
+      - name: Lint (clippy)
+        run: cargo clippy --all-features -- -D warnings
```

Comment:
should keep this disabled

Reply:
disabled in [96f903a](https://github.com/zksecurity/noname/pull/100/commits/96f903af9776c17cc785dadafc2d33382d02856d)